administrators.rst

```diff
@@ -236,11 +236,7 @@ Fields:
 Dataverse
 ^^^^^^^^^
 
-As of Storage Service version 0.9, Dataverse integration is a beta feature. It
-requires the use of a development branch of Archivematica
-(https://github.com/artefactual/archivematica/tree/dev/issue-8693-dataverse) and
-of the Automation Tools
-(https://github.com/artefactual/automation-tools/tree/dev/dataverse)
+Dataverse Integration is supported with Archivematica v1.8+ and Storage Service v0.13+
```

Comment:
Instead of `+` can you use `and higher` or similar? I think the meaning is clear but let's avoid any ambiguity :)

Reply:
yes, make sense.